test-foundry/TestStessHeapOrdering.t.sol

```diff
@@ -14,30 +14,30 @@ contract HeapStorage {
     function setUp() public {
         for (uint256 i = 0; i < TESTED_SIZE; i++) {
```

Comment:
perhaps just change here instead of casting below

Reply:
done